### ISSUE TITLE
Wagtail editor on non-admin pages?

### ISSUE THREAD

**Issue by timonweb** [wagtail/wagtail#5557]
Is there an easy way to use Wagtail editor (draftail powered) on non-admin pages? I hacked my own solution with raw Draftail (https://www.draftail.org/), but it would be nice to be able to re-use the built-in version of Draftail. 

Currently, it seems that Wagtail editor is tightly coupled with the admin section and I had to jump through a few hoops to bring it to a non-admin page.

**Comment by timonweb** [comment]
Hey, maybe it sounds like a question, but my idea was to check if someone has solved this and if not, then if any help is needed.

**Comment by TorselloM** [comment]
@timonweb Did you find a solution ? Kinda stuck here aswell. cheers

**Comment by TorselloM** [comment]
Hey @timonweb, did you find a solution ? Kinda stuck here ...

**Comment by timonweb** [comment]
@TorselloM yes I rolled my own solution here: https://github.com/timonweb/non-admin-draftail, it doesn't have image/doc uploads working but it works. Currently I'm in the process of rewriting it and soon will release a more polished solution with image and document upload working. Stay tuned.

**Comment by TorselloM** [comment]
@timonweb Yeah I found it after asking, kept the question up so you could answer and maybe other people will be helped. Anyway great work and thanks a lot, will use your solution !

**Comment by timonweb** [comment]
I'm finally done with the major re-write, the updated v. 0.9.0 is here: https://github.com/timonweb/non-admin-draftail let me know in the issues if you have questions.
